core/templates/dev/head/dashboard/notifications_dashboard.html

```diff
@@ -37,11 +37,13 @@ <h2 class="oppia-page-heading">Notifications</h2>
           <div class="oppia-notification-content">
             <span class="oppia-notification-type">
               <span ng-if="notification.type === 'feedback_thread'"
-                    class="glyphicon glyphicon-comment"
-                    title="Feedback thread"></span>
+                    title="Feedback thread">
```

Comment:
Maybe this line can go up as we have some space now. Incorrect indentation (line 41).